app/renderer/components/config_menu.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
                         cancelButtonText: "No, cancel import"
                     });
                     confirmationAlert.toggle().then(() => {
+                        ('#config-modal').modal('close');
```

Comment:
This is a materialize method, in this project (to avoid [ñapa code](https://calidadysoftware.wordpress.com/2015/07/10/naas-napa-as-a-service/)) I'm wrapping all materialize crap inside the file `materialize.js`

In fact, there is already a wrapped method for this (closeModals). Please use that method instead